platform/security/idp/security-idp-client/src/main/java/org/codice/ddf/security/idp/client/LogoutRequestService.java

```diff
@@ -236,6 +258,104 @@ private Response getSamlpRedirectLogoutRequest(String relayState, LogoutRequest
     return ok.build();
   }
 
+  private String extractSubject(HttpSession httpSession) {
+    return Stream.of(httpSession.getAttribute(SecurityConstants.SAML_ASSERTION))
+        .map(SecurityTokenHolder.class::cast)
```

Comment:
❓ Do we not want to filter on isInstance first? (even just to protect from wierd bundle reloading errors?)

Reply:
Adding.